client/packages/system/src/Patient/DetailView/ProgramEnrolment/ListView.tsx

```diff
@@ -73,7 +74,11 @@ const ProgramListComponent: FC = () => {
         setDocumentName(row.document.name);
         setCurrent(PatientModal.Program);
       }}
-      noDataElement={<NothingHere />}
+      noDataElement={
+        <Box width={'100%'} paddingTop="100">
+          <NothingHere />
+        </Box>
+      }
```

Comment:
Is there a better way to do this? The Noting Here box was sticking to the top and under the drop shadow of the tab panel...

Reply:
yes - adding `height: '100%',` to the `StyledTabContainer` in the `PatientTab` will allow the grandparent to be full height, which allows the flex element (NothingHere) to be vertically positioned correctly.

Reply:
Thanks that works better!